packages/cloud_functions/cloud_functions_platform_interface/lib/src/method_channel/method_channel_https_callable.dart

```diff
@@ -2,31 +2,29 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-// @dart=2.9
-
 import 'dart:async';
 
-import 'package:cloud_functions_platform_interface/cloud_functions_platform_interface.dart';
-import 'package:cloud_functions_platform_interface/src/method_channel/method_channel_firebase_functions.dart';
+import '../../cloud_functions_platform_interface.dart';
+import 'method_channel_firebase_functions.dart';
 
 import 'utils/exception.dart';
 
 /// Method Channel delegate for [HttpsCallablePlatform].
 class MethodChannelHttpsCallable extends HttpsCallablePlatform {
   /// Creates a new [MethodChannelHttpsCallable] instance.
-  MethodChannelHttpsCallable(FirebaseFunctionsPlatform functions, String origin,
-      String name, HttpsCallableOptions options)
+  MethodChannelHttpsCallable(FirebaseFunctionsPlatform? functions,
+      String? origin, String name, HttpsCallableOptions? options)
       : super(functions, origin, name, options);
 
   @override
   Future<dynamic> call([dynamic parameters]) {
     return MethodChannelFirebaseFunctions.channel
         .invokeMethod('FirebaseFunctions#call', <String, dynamic>{
-      'appName': functions.app.name,
+      'appName': functions!.app!.name,
```

Comment:
why is `functions` nullable if it's to use `!`?

Reply:
Yup - at this point `functions` should never be null. Also, `options` should also never be `null` since the public package should be providing a default (which is does).

Reply:
Done